docs_src/OVMS/runObjectDetectionPipelineWithNewModel.md

```diff
@@ -1,74 +1,81 @@
 2. Download new model
 3. Update environment variables of detection pipeline for new model
-4. rebuild containers
+4. Build and Run
 
 
 ## Add New Section to Config File for Model Server
+
 Here is the config file location: `configs/opencv-ovms/models/2022/config_template.json`, edit the file and append the following configuration section template
 ```json
 ,
-{"config": {
-                "name": "yourModelName",
-                "base_path": "/models/yourModelName/FP16-INT8",
-                "nireq": 1,
-                "batch_size":"1",
-                "shape": "(1,608,608,3)",
-                "layout": "NHWC:NCHW",
-                "plugin_config": {"PERFORMANCE_HINT": "LATENCY"},
-                "target_device": "{target_device}"},
-                "latest": { "num_versions": 1 }
-        }
+{
+      "config": {
+        "name": "ssd_mobilenet_v1_coco",
+        "base_path": "/models/ssd_mobilenet_v1_coco/FP32",
+        "nireq": 1,
+        "batch_size": "1",
+        "plugin_config": {
+          "PERFORMANCE_HINT": "LATENCY"
+        },
+        "target_device": "{target_device}"
+      },
+      "latest": {
+        "num_versions": 1
+      }
+    }
 ```
-!!! Note
-    `shape` is optional and takes precedence over batch_size, please remove this attribute if you don't know the value for the model.
-
 !!! Note
     Please leave `target_device` value as it is, as the value `{target_device}` will be recognized and filled out by script run.
 
-!!! Note
-    Please update value of `name` to your model name
-
-!!! Note
-    `base_path`: make sure the "yourModelName" is same as the `name` value
-
-You can find each parameter description from model server [ovms_docs_parameters](https://docs.openvino.ai/2023.0/ovms_docs_parameters.html).
+You can find the parameter description in the [ovms docs](https://docs.openvino.ai/2023.1/ovms_docs_parameters.html).
 
 ## Download New Model
+
 The pipeline run script automatically download the model files if it is part of [open model zoo supported list](https://github.com/openvinotoolkit/open_model_zoo/blob/master/demos/object_detection_demo/python/models.lst); otherwise, please add your model files manually to `configs/opencv-ovms/models/2022/`. When you add your model manually, make sure to follow the model file structure as <model_name>/<Precision>/1/modelfiles, for example:
 
 ```text
-instance-segmentation-security-1040
-│   └── FP16-INT8
-│       └── 1
-│           ├── instance-segmentation-security-1040.bin
-│           └── instance-segmentation-security-1040.xml
+ssd_mobilenet_v1_coco
+├── FP32
+│   └── 1
+│       ├── ssd_mobilenet_v1_coco.bin
+│       └── ssd_mobilenet_v1_coco.xml
+└── ssd_mobilenet_v1_coco_2018_01_28
+    ├── 1
+    ├── checkpoint
+    ├── frozen_inference_graph.pb
+    ├── model.ckpt.data-00000-of-00001
+    ├── model.ckpt.index
+    ├── model.ckpt.meta
+    ├── pipeline.config
+    └── saved_model
+        ├── saved_model.pb
+        └── variables
```

Comment:
why this `ssd_mobilenet_v1_coco_2018_01_28` is a sub-directory of ssd_mobilenet_v1_coco? and do we need this folder?